flake.nix

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+{
+  description = "The tezos-packaging flake";
+
+  inputs = {
+    nixpkgs.url = "github:serokell/nixpkgs/be220b2dc47092c1e739bf6aaf630f29e71fe1c4";
+
+    flake-utils.url = "github:numtide/flake-utils";
+
+    serokell-nix.url = "github:serokell/serokell.nix/46d762e5107d10ad409295a7f668939c21cc048d";
+
+    flake-compat.url = "github:edolstra/flake-compat";
+    flake-compat.flake = false;
+
+    opam-nix.url = "github:tweag/opam-nix/44b97c71a379bb9517135600c7d5e83d941e65d9";
+
+    opam-repository.url = "gitlab:tezos/opam-repository/845375388d477153b36344f69a23a1b34b2bf82e";
+    opam-repository.flake = false;
+
+    tezos.url = "gitlab:tezos/tezos/cb9f439e58c761e76ade589d1cdbd2abb737dc68";
+    tezos.flake = false;
+  };
+
+  outputs = inputs@{ self, nixpkgs, flake-utils, ... }: {
+
+      nixosModules = {
+        tezos-node = import ./nix/modules/tezos-node.nix;
+        tezos-accuser = import ./nix/modules/tezos-accuser.nix;
+        tezos-baker = import ./nix/modules/tezos-baker.nix;
+        tezos-signer = import ./nix/modules/tezos-signer.nix;
+      };
+
+    } // flake-utils.lib.eachSystem [
+      "x86_64-linux"
+    ] (system:
+    let
+
+      ocaml-overlay = callPackage ./nix/build/ocaml-overlay.nix {};
+
+      pkgs = import nixpkgs {
+        inherit system;
+        overlays = [ inputs.serokell-nix.overlay ocaml-overlay ];
+      };
+
+      callPackage = pkg: input:
+        import pkg (inputs // { inherit sources protocols meta pkgs; } // input);
+
+      protocols = pkgs.lib.importJSON ./protocols.json;
+
+      meta = pkgs.lib.importJSON ./meta.json;
+
+      sources = { inherit (inputs) tezos opam-repository; };
+
+      binaries = callPackage ./nix {};
+
+      tezos-release = callPackage ./release.nix {};
+
+    in rec {
+
+      legacyPackages = pkgs;
+
+      inherit tezos-release;
+
+      apps.tezos-client = {
```

Comment:
Why only `tezos-client`? :thinking: 

Are these `apps` some default flake outputs?

Reply:
apps is thing to run with nix run, do we need it at all?

Reply:
I don't think that we really need it